src/ClientHandle.cpp

```diff
@@ -2568,6 +2586,42 @@ void cClientHandle::SendDisplayObjective(const AString & a_Objective, cScoreboar
 
 
 
+void cClientHandle::SendSetSubTitle(const cCompositeChat & a_SubTitle)
```

Comment:
That is actually incorrect. I managed to export the functions to the lua API and I was testing some of them the other day. It appears that `SendSetTitle(...)` and `SendSetSubTitle(...)` both when called (doesn't matter which one) will display the specifed type. `SendTitleTimes(...)` is just a preliminary step to displaying the title or subtitle. If both the title and the subtitle packet are sent at the same time they will both appear together.